crates/neon-runtime/src/napi/bigint.rs

```diff
@@ -0,0 +1,57 @@
+use crate::napi::bindings as napi;
+use crate::raw::{Env, Local};
+use std::mem::MaybeUninit;
+
+/// Create a new bigint object
+///
+/// # Safety
+///
+/// `env` is a raw pointer. Please ensure it points to a napi_env that is valid for the current context.
+pub unsafe fn new_bigint(env: Env, value: i64) -> Local {
+    let mut local = MaybeUninit::zeroed();
+    let status = napi::create_bigint_int64(env, value, local.as_mut_ptr());
+    assert_eq!(status, napi::Status::Ok);
```

Comment:
I haven't checked to see if this can ever throw an exception--which is necessary for this `assert` to be safe. Can you double check?

Reply:
@kjvalencik do you have some guidance on how to determine this? The [napi docs for the function](https://nodejs.org/api/n-api.html#napi_create_bigint_int64) do not mention exceptions -- do I need to check the nodejs or v8 source code to determine this?

Reply:
Unfortunately, it's not super clear. It's a manual task that's a combination of reading the Node/Node-API source and sometimes the JS spec.

I expect that this *can't* ever fail except with bad data (which Rust type system protects against).